Source/GSStandardWindowDecorationView.m

```diff
@@ -97,7 +97,7 @@ - (void) updateRects
 
   if (hasTitleBar)
     {
-      CGFloat titleHeight = [theme titlebarHeight];
+      CGFloat titleHeight = [[NSScreen mainScreen] userSpaceScaleFactor] * [theme titlebarHeight];
```

Comment:
Are you sure that this second usage is also needed? According to my view we would only need the manual scaling for coordinates in window coordinate systems. The values here are in the bounds coordinate system and should be automatically scaled. If this does not happen, I will have to rethink a few things.

Reply:
Ah, I’m not actually - I’ll remove this and give it another test.

Reply:
It does work fine with just the first change, I've amended my commit - thanks for the help.